boards/xtensa/esp32_ethernet_kit/Kconfig.defconfig

```diff
@@ -8,7 +8,7 @@ config BOARD
 	depends on BOARD_ESP32_ETHERNET_KIT
 
 config ESP_SPIRAM
```

Comment:
Probably removing this at all and enabling ESP_SPIRAM elsewhere, on the application level would be a better solution.

Reply:
Shouldn't we allow the user to use all available memory by default?